wp-content/themes/hurumap/functions.php

```diff
@@ -286,3 +298,27 @@ function custom_preview_page_link($link) {
     }
     return $paths;
 });
+
+/**
+ * Redirect to thumbnail by Id !
+ *
+ * @param array $data Options for the function.
+ */
+function redirect_to_media_by_id( $data ) {
+    $media = get_post_resource( $data["id"] ,'media');
+ 
+  if ( empty( $media ) ) {
+    return null;
+  }elseif( empty($media['id'])){
+    return null;
+}
+  wp_redirect( $media["source_url"] );
+  exit;
+}
```

Comment:
Format & just for my understanding, what's the difference between
```php
  wp_redirect( $media["source_url"] );
  exit;
```

and 

```php
return wp_redirect( $media["source_url"] );
```
?